CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+## v0.32.1
+
+*July 12, 2019*
+
+Special thanks to external contributors on this release: 
+@ParthDesai, @climber73, @jim380, @ashleyvega
+
+This release contains breaking changes to our libs folder and ABCI namely:
+- Add a "Recheck Tx" indicator 
+- Removal of `db/debugDB`, `common/colors.go`, `errors/errors.go`, `common/heap.go`, `date.go`, `io.go` and
+  `GoPath()`, `Prompt()` and `IsDirEmpty()` functions from `os.go`
+
+Friendly reminder, we have a [bug bounty
+program](https://hackerone.com/tendermint).
+
+### BREAKING CHANGES:
+
+- CLI/RPC/Config
+
+- Apps
+
+- Go API
+
+  - [abci] [\#2127](https://github.com/tendermint/tendermint/issues/2127) ABCI / mempool: Add a "Recheck Tx" indicator. Breaks the ABCI
```

Comment:
The breaking change here should be described more clearly as introducing the RequestCheckTx and RequestDeliverTx types as the args for those methods in the `abci.Application` interface. Ie. this is specifically about the breaking change to the Go API.

Adding the CheckTxType field is something that should be logged as an `[abci]` feature. It also corresponds to a bump in the ABCI version (see comment below).